README.md

```diff
@@ -237,6 +275,17 @@ Finally, we use `host` `network_mode`, this is because our server code refers to
 
 We require our Params FTP Server to be running before starting the prover node. The prover node must copy the parameters from the FTP server to it's own volume to operate correctly.
 
+Make sure you had reviewed the [Prover Node Configuration](#prover-node-configuration) part and changed the config files.
```

Comment:
I assume this "Quick start" section is for new prover nodes.
So maybe need make sure, if the hoster had installed everything (docker, cuda toolkit) and finished the config, run bash scripts/upgrade.sh and bash scripts/start.sh can work for it.

This is because I didn't see any checking or error treating inside scripts/upgrade.sh, so not sure if it is a fresh env, whether it will work well.

Reply:
i have tested for fresh env and it is fine